cardinal/server/server_test.go

```diff
@@ -61,6 +62,33 @@ func makeTestTransactionHandler(t *testing.T, world *ecs.World, opts ...Option)
 	}
 }
 
+func TestIfServeSetEnvVarForPort(t *testing.T) {
+	world := inmem.NewECSWorldForTest(t)
+	alphaTx := ecs.NewTransactionType[SendEnergyTx, SendEnergyTxResult]("alpha")
+	betaTx := ecs.NewTransactionType[SendEnergyTx, SendEnergyTxResult]("beta")
+	gammaTx := ecs.NewTransactionType[SendEnergyTx, SendEnergyTxResult]("gamma")
+	assert.NilError(t, world.RegisterTransactions(alphaTx, betaTx, gammaTx))
```

Comment:
```suggestion
	assert.NilError(t, world.RegisterTransactions(alphaTx))
```

will be fine with one tx registered